lib/db/schema/patch-048-049.sql

```diff
@@ -0,0 +1,47 @@
+CREATE PROCEDURE `prune_3` (IN `olderThan` BIGINT UNSIGNED)
+BEGIN
+  SELECT @lockAcquired := GET_LOCK('fxa-auth-server.prune-lock', 3);
+
+  IF @lockAcquired THEN
+    DELETE FROM accountResetTokens WHERE createdAt < olderThan ORDER BY createdAt LIMIT 10000;
+    DELETE FROM passwordForgotTokens WHERE createdAt < olderThan ORDER BY createdAt LIMIT 10000;
+    DELETE FROM passwordChangeTokens WHERE createdAt < olderThan ORDER BY createdAt LIMIT 10000;
+    DELETE FROM unblockCodes WHERE createdAt < olderThan ORDER BY createdAt LIMIT 10000;
+    DELETE FROM signinCodes WHERE createdAt < olderThan ORDER BY createdAt LIMIT 10000;
+
+    SELECT RELEASE_LOCK('fxa-auth-server.prune-lock');
+  END IF;
+END;
+
+DROP PROCEDURE `expireSigninCodes_1`;
```

Comment:
We don't, but this one is a slightly different case to normal. We're not introducing a new version of it, we're dropping it because it was never used and updating a different stored procedure to do what we want instead.

It's completely safe but also I'm fine if we don't want to do it like this due to convention or whatever.